float64[] radi  # radius for landmarks
float64[] x_pose # x position for landmarks
float64[] y_pose  # y position for landmarks
Header header # Header for msg
